cfme/tests/services/test_rest_services.py

```diff
@@ -203,6 +204,22 @@ def cart(appliance, delete_carts):
         cart.action.delete()
 
 
+@pytest.fixture
+def update_advanced_settings(appliance):
```

Comment:
Looks good , Please add a doc block explaining this a bit like what is the default setting .